01_todo_app/web/src/components/landing/Footer.jsx

```diff
@@ -0,0 +1,138 @@
+                Footer
+            </h2>
+            <div className="xl:grid xl:grid-cols-9 xl:gap-8 mt-8">
+                <div className="space-y-8 col-span-3">
+                    <img className="h-14 -ml-2" src="golem_create.png" alt="Golem Factory" />
+                    <p className="text-sm leading-6 text-white">
+                        Golem Network is a decentralized computing marketplace that supports Web3 in a most efficient way possible
+                    </p>
+                    <div className="flex space-x-6">
+                        {navigation.social.map((item) => (
+                            <a key={item.name} href={item.href} target="_blank" className="text-gray-600 hover:text-gray-500">
+                                <span className="sr-only">{item.name}</span>
+                                <item.icon className="h-7 w-7" aria-hidden="true" />
+                            </a>
+                        ))}
+                    </div>
+                </div>
+                <div className="col-span-1"></div>
+                <div className="mt-16 grid grid-cols-12 gap-8 xl:mt-0 col-span-5">
+                    <div className="grid gap-y-8 xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 col-span-12">
+                        <div>
+                            <h3 className="text-sm font-semibold leading-6 text-white">Developers</h3>
+                            <ul role="list" className="mt-6 space-y-4">
+                                {navigation.first.map((item) => (
+                                    <li key={item.name}>
+                                        <a href={item.href} target="_blank" className="text-sm leading-6 text-gray-500 hover:text-gray-300">
+                                            {item.name}
+                                        </a>
+                                    </li>
+                                ))}
+                            </ul>
+                        </div>
+                        <div className="">
+                            <h3 className="text-sm font-semibold leading-6 text-white">Community</h3>
+                            <ul role="list" className="mt-6 space-y-4 col-span-4">
+                                {navigation.second.map((item) => (
+                                    <li key={item.name}>
+                                        <a href={item.href} target="_blank" className="text-sm leading-6 text-gray-500 hover:text-gray-300">
+                                            {item.name}
+                                        </a>
+                                    </li>
+                                ))}
+                            </ul>
+                        </div>
+                        <div className="mt-10 md:mt-0">
+                            <h3 className="text-sm font-semibold leading-6 text-white">Resources</h3>
+                            <ul role="list" className="mt-6 space-y-4 col-span-4">
```

Comment:
Exercise to skill-up using components: Extract this to `FooterLinkList` component which should reside in it own file.